### ISSUE TITLE
[🐛 bug report] Cornell box scene doesn't work with double precision variants

### ISSUE THREAD

**Issue by leroyvn** [mitsuba-renderer/mitsuba3#200]
## Summary

The dict returned by `mi.cornell_box()` cannot be loaded with double-precision variants.

## System configuration

- Platform: Ubuntu 20.04
- Compiler: clang 11
- Python version: 3.10.5
- Mitsuba 3 version: 3.0.1
- Compiled variants:
    * `scalar_rgb`
    * `scalar_rgb_double`

## Description

When creating a default scene with `mi.cornell_box()`, the load step raises a `RuntimeError`:

```
RuntimeError: ​[xml_v.cpp:335] Unkown value type: <class 'mitsuba.scalar_rgb.ChainTransform4f'>
```

I assume this is due to [`cornell_box()` using `mi.scalar_rgb.ScalarTransform4f` rather than `mi.ScalarTransform4f`](https://github.com/mitsuba-renderer/mitsuba3/blob/847686bb68fdcdc0b7c483d6e09b1245499fbdda/src/python/python/util.py#L543). 

## Steps to reproduce

Hereafter is a MWE (requires the `scalar_rgb_double` variant):

```python
import mitsuba as mi

mi.set_variant("scalar_rgb_double")
scene_dict = mi.cornell_box()
scene = mi.load_dict(scene_dict)
```

**Comment by Speierers** [comment]
Thanks for reporting this @leroyvn. It will be fixed in the next release: [d3bc493](https://github.com/mitsuba-renderer/mitsuba3/commit/d3bc493527e638617c96b2535d30a044a9273c80)

**Comment by leroyvn** [comment]
Thanks @Speierers!
